src/main/java/com/sonymobile/jenkins/plugins/lenientshutdown/cli/LenientQuietDownCommandBase.java

```diff
@@ -0,0 +1,119 @@
+/*
+ *  The MIT License
+ *
+ *  Copyright (c) 2014 Sony Mobile Communications Inc. All rights reserved.
```

Comment:
I'm guessing this is a bit of a copy from `LenientQuietDownCommand` so the old copy should in that case stay, but significant changes by you so adding your copy information as well is custom.